packages/cadl-ranch-specs/http/encode/bytes/mockapi.ts

```diff
@@ -85,23 +85,23 @@ Scenarios.Encode_Bytes_Header_base64url = passOnSuccess(createHeaderMockApis("ba
 Scenarios.Encode_Bytes_Header_base64urlArray = passOnSuccess(createHeaderMockApis("base64url-array", "dGVzdA,dGVzdA"));
 
 // Request body
-Scenarios.Encode_Bytes_RequestBody_default = passOnSuccess(createRequestBodyMockApis("default", "dGVzdA=="));
+Scenarios.Encode_Bytes_RequestBody_default = passOnSuccess(createRequestBodyMockApis("default", '"dGVzdA=="'));
 Scenarios.Encode_Bytes_RequestBody_octetStream = passOnSuccess(
   createRequestBodyMockApis("octet-stream", pngFile, "application/octet-stream"),
 );
 Scenarios.Encode_Bytes_RequestBody_customContentType = passOnSuccess(
   createRequestBodyMockApis("custom-content-type", pngFile, "image/png"),
 );
```

Comment:
This validation may not work. It seems comparing bytes of png file to raw content (which is base64 encoded).

Not sure about this, so I didn't change.